src/i18n/en.json

```diff
@@ -168,31 +171,38 @@
     "doGoToAccountManagement": "Go to account management",
     "authenticationMethod": "Authentication method",
     "primaryAddress": "Primary address",
-    "permanentAddress": "EN: Vakinainen osoite",
-    "temporaryAddress": "EN: Väliaikainen osoite",
-    "permanentForeignAddress": "EN: Vakinainen ulkomainen osoite",
+    "permanentAddress": "Permanent address",
+    "temporaryAddress": "Temporary address",
+    "permanentForeignAddress": "Permanent foreign address",
+    "primaryEmail": "Primary email",
     "primaryPhone": "Primary phone number",
     "primaryAddressAriaDescription": "This is your primary address",
+    "primaryEmailAriaDescription": "This is your primary email",
     "primaryPhoneAriaDescription": "This is your primary phone number",
-    "verifiedDataInformationPrefix": "EN: Merkityt",
-    "verifiedDataInformationAriaPrefix": "EN: Alla olevat",
-    "verifiedDataInformation": "EN: <0></0> tiedot on vahvistettu virallisesta väestötietojärjestelmästä, eikä niitä ei voi tämän takia muokata Helsinki Profiilissa. Voit tutustua halutessasi väestötietojärjestelmään tallennettuihin tietoihisi suomi.fi verkkopalvelussa: <1>siirry suomi.fi palveluun </1>",
+    "verifiedDataInformationPrefix": "Marked",
+    "verifiedDataInformationAriaPrefix": "Those below",
+    "verifiedDataInformation": "<0></0> the data has been verified from the official Population Information System and they can thus not be edited in the Helsinki Profile. You can check your data in the Population Information System through the suomi.fi web service: <1>go to the suomi.fi service </1>",
```

Comment:
Isn't it so that the `<0></0>` in `verifiedDataInformation` is replaced with either `verifiedDataInformationPrefix` or `verifiedDataInformationAriaPrefix`? Doing those replacements doesn't produce very good sentences.

It's not a good idea to build sentences in this way at all, as it will produce bad results in some language(s). There should probably be different translations for the visible case and the aria case, even if they would contain mostly the same words.

Reply:
Yes, this is ugly and quite unreadable. In this case React components are inserted in to the translations:
```
<Trans
        i18nKey="profileInformation.verifiedDataInformation"
        components={[
          forAria
            ? t('profileInformation.verifiedDataInformationAriaPrefix')
            : t('profileInformation.verifiedDataInformationPrefix'),
          <NewWindowLinkWithChildrenAsTitle
            link={t('profileInformation.verifiedDataInformationLink')}
            tabIndex={forAria ? 0 : -1}
          />,
        ]}
      />
```
Components are inserted by index, that's why there is <0>, <1> This is the way i18next have [documented](https://react.i18next.com/latest/trans-component) the usage.

Looks like there is also newer way of using keys instead of indexes. This usage was copied from the usage of another translation (profileForm.terms) which dates before keys could be used.

Reply:
This PR unifies the translation files, so same files can be used in master and develop branches. Switching from indexes to keys, would require code changes in both branches, cherry-picked of course. But not in this PR, perhaps a new ticket for this?

Reply:
> not in this PR, perhaps a new ticket for this?

That works.

Reply:
In this case the same result can be achieved with both the indices and the keys. The main problem here is probably that the translator didn't have enough context how the translated strings are used. So this is not that much a technical problem but a communication problem.

There are (at least) two options to solve that:
1. communicate more/better
2. reduce the need for communication, i.e. make the translator's job easier

I think point 2 should be tried first and only if that isn't sufficient, then use point 1.

Here's a quick sketch of a diff what the translation file could look like so that it doesn't have to communicate that much context to the translator:

```diff
-"verifiedDataInformationPrefix": "EN: Merkityt",
-"verifiedDataInformationAriaPrefix": "EN: Alla olevat",
-"verifiedDataInformation": "EN: <0></0> tiedot on vahvistettu virallisesta väestötietojärjestelmästä, eikä niitä ei voi tämän takia muokata Helsinki Profiilissa. Voit tutustua halutessasi väestötietojärjestelmään tallennettuihin tietoihisi suomi.fi verkkopalvelussa: <1>siirry suomi.fi palveluun </1>",
+"verifiedDataInformation": "EN: Merkityt tiedot on vahvistettu virallisesta väestötietojärjestelmästä, eikä niitä ei voi tämän takia muokata Helsinki Profiilissa. Voit tutustua halutessasi väestötietojärjestelmään tallennettuihin tietoihisi suomi.fi verkkopalvelussa: <0>siirry suomi.fi palveluun </0>",
+"verifiedDataInformationAria": "EN: Alla olevat tiedot on vahvistettu virallisesta väestötietojärjestelmästä, eikä niitä ei voi tämän takia muokata Helsinki Profiilissa. Voit tutustua halutessasi väestötietojärjestelmään tallennettuihin tietoihisi suomi.fi verkkopalvelussa: <0>siirry suomi.fi palveluun </0>",
```

Reply:
I agree, if translation process needs better communication, there is a bigger problem beneath. :)